android/src/main/java/app/notifee/core/NotificationManager.java

```diff
@@ -483,7 +483,7 @@
 
                 if (notificationType != NOTIFICATION_TYPE_TRIGGER) {
                   // Cancel notifications displayed by FCM
-                  if (tag != null && tag.startsWith("FCM-") == true) {
+                  if (tag != null && id.equals("0")) {
```

Comment:
Thoughts on maybe a code comment what this 0 is about for other contributors 😅

Reply:
good idea! will update with more details. I've also in this p/r linked the original issue